### PR TITLE
React routing: Route Added

### PR DESCRIPTION
**Pull request by mdarif6**
- Routing Added to all pages
- Link provided by replacing anchor tag

### CODE REVIEW

File: src/pages/products-page/HeaderHome.jsx [hunk 1]
```diff
@@ -22,24 +23,25 @@ export default function HeaderHome() {
         </div>
         <div className="header-icons">
           <div className="ha-nav-icons">
-            <a href="/login_page/login_page.html">
+            <Link to="/login">
```
Comment (Shah-Arsalan) [inline src/pages/products-page/HeaderHome.jsx:26]: You have used link in multiple places . Using useNavigate would be a better idea.

**Review by Shah-Arsalan — APPROVED** [review]

File: src/pages/products-page/HeaderHome.jsx (hunk 1 above)
Comment (mdarif6) [reply src/pages/products-page/HeaderHome.jsx:26]: okay will check in next PR during adding functionality

File: src/pages/products-page/AsideFilter.jsx [hunk 2]
```diff
@@ -21,18 +21,34 @@ export default function AsideFilter() {
...
-        <br />
-        <input type="checkbox" name="category3" id="category3" value="kids" />
-        <label for="category3">Kids</label>
-        <br />
-        <input type="checkbox" name="category4" id="category4" value="indie" />
-        <label for="category4">Inide</label>
-        <br />
+        <div>
+          <input type="checkbox" name="category1" id="category1" value="men" />
+          <label for="category1">Men</label>
```
Comment (samyakshah3008) [inline src/pages/products-page/AsideFilter.jsx:26]: Good semantic tags

**Review by samyakshah3008 — APPROVED** [review]
Good folder structure, clean code ✅ 

Small suggestion: Can add comments where necessary 
Also in the preview description you can provide live link or attach gif for better review experience 

Good for approval ✅

**Pull request merged**